inbc-program/inbc/parser/parser.py

```diff
@@ -65,11 +65,11 @@ def parse_source_args(self) -> None:
 
         # Application Add Command
         app_add_parser = app_subparsers.add_parser('add')
-        app_add_parser.add_argument('--gpgKeyUri', '-gku', required=True,
+        app_add_parser.add_argument('--gpgKeyUri', '-gku', required=False,
                                     type=lambda x: validate_string_less_than_n_characters(
                                         x, 'str', 1500),
                                     help='Uri from which to download GPG key')
-        app_add_parser.add_argument('--gpgKeyName', '-gkn', required=True,
+        app_add_parser.add_argument('--gpgKeyName', '-gkn', required=False,
                                     type=lambda x: validate_string_less_than_n_characters(
                                         x, 'str', 200),
                                     help='Name to store the GPG key information')
```

Comment:
**Suggestion:** The 'required' flag for '--gpgKeyUri' and '--gpgKeyName' arguments in the 'app_add_parser' has been changed from True to False. Ensure that the rest of the code properly handles cases where these arguments might not be provided, to prevent runtime errors. [possible issue]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        # It's assumed that the application_add function is designed to handle optional GPG key parameters.
        # If not, the function should be updated accordingly to avoid runtime errors when these arguments are not provided.
```